site/content/component/mqtt-adapter.md

```diff
@@ -201,30 +201,50 @@ The same holds true analogously for the *hono-service-device-registry.hono* addr
 
 ## Using the Telemetry Topic Hierarchy
 
-The following examples use a username and password for connecting to MQTT. The username reflects the auth-id of the [Credentals API]({{< relref "api/Credentials-API.md" >}}) and usually represents a device.
+The following examples use a username and password for connecting to MQTT. The username reflects the auth-id of the [Credentals API]({{< relref "api/Credentials-API.md" >}}) and usually represents a device. It may be identical to the device-id.
 
-**NB** The standard setup of the [Device Registry component]({{< relref "component/device-registry.md" >}}) provides the example auth-id `sensor1` with the password `hono-secret` which are used here. 
+**NB** The standard setup of the [Device Registry component]({{< relref "component/device-registry.md" >}}) provides the example auth-id `sensor1` with the password `hono-secret` 
+and the device-id `4711` which are used here. Some devices may not know their device-id but only their auth-id - this case is covered in the examples as well. 
 
 
-### Upload Telemetry Data
+### Upload Telemetry Data (authenticated devices)
 
-* Topic: `telemetry/${tenantId}/${deviceId}`
-* Client-id: ${deviceId}
+* Topic: `telemetry/${tenant-id}/${device-id}` or just `telemetry` (short form)
+* Client-id: not specified, use e.g. ${device-id} or any unique identifier 
```

Comment:
just remove this line, otherwise people might think they need to set it explicitly while most (if not all) MQTT clients will set this to a generated value if not set